apps/analysis/views.py

```diff
@@ -113,6 +115,30 @@ def get_queryset(self):
         )
 
 
+class DiscardedEntriesViewSet(viewsets.ModelViewSet):
```

Comment:
Let's use AnalysisPillarDiscardedEntriesViewset
```suggestion
class AnalysisPillarDiscardedEntriesViewset(viewsets.ModelViewSet):
```